src/bpftrace.cpp

```diff
@@ -107,6 +107,7 @@ int BPFtrace::add_probe(ast::Probe &p)
       probe.path = "/proc/self/exe";
       probe.attach_point = "BEGIN_trigger";
       probe.type = probetype(attach_point->provider);
+      probe.log_size = this->log_size_;
```

Comment:
```suggestion
      probe.log_size = log_size_;
```
Just to match the style we use throughout the codebase

Reply:
fixed in latest commit